ChattoAdditions/Source/Input/ChatInputBarPresenter.swift

```diff
@@ -191,7 +191,7 @@ extension BasicChatInputBarPresenter {
     }
 }
 
-private class InputContainerView: UIInputView {
```

Comment:
Can you extract this class into a separate file? Now it's used in several places